pyvortex/src/scalar.rs

```diff
@@ -14,7 +14,7 @@ use vortex_error::vortex_panic;
 use vortex_scalar::{PValue, Scalar, ScalarValue};
 
 pub fn scalar_into_py(py: Python, x: Scalar, copy_into_python: bool) -> PyResult<PyObject> {
-    let (value, dtype) = x.into_parts();
+    let (dtype, value) = x.into_parts();
```

Comment:
👀

Reply:
lol i guess dan wrote the same method at the same time as me we just wrote the args in a different order

Reply:
i just sort of yolo resolved the conflict and this was the outcome